tasks/opsep/update-l2pao-to-aliased-l1pao/SignFromJson.s.sol

```diff
@@ -0,0 +1,188 @@
+    address constant batchInboxAddress = 0xff00000000000000000000000000000011155420; // In registry yaml.
+    
+    Types.ContractSet proxies;
+
+    // This gives the initial fork, so we can use it to switch back after fetching data.
+    uint256 initialFork;
+
+    /// @notice Sets up the contract
+    function setUp() public {
+        proxies = _getContractSet();
+
+        // Fetch variables that are not expected to change from an older block.
+        initialFork = vm.activeFork();
+        vm.createSelectFork(vm.envString("ETH_RPC_URL"), 5705332); // This block is from April 15 2024 at 11:40am PT.
+
+        require(unaliasedL1PAO == undoL1ToL2Alias(aliasedL1PAO), "setUp-100");
+        require(aliasedL1PAO == applyL1ToL2Alias(unaliasedL1PAO), "setUp-101");
+
+        vm.selectFork(initialFork);
+    }
+
+    function checkL2PAOSafe() internal view {
+        console.log("Running assertions on the L2PAO Safe");
+
+        address[] memory l2paoSafeOwners = l2paoSafe.getOwners();
+        require(l2paoSafeOwners.length == 1, "checkL2PAOSafe-100");
+        require(l2paoSafeOwners[0] == l2PaoSafeEOA, "checkL2PAOSafe-101"); // stays the same
+        require(l2paoSafe.getThreshold() == 1, "checkL2PAOSafe-102");
+    }
+
+    function checkL2PA() internal view {
+        console.log("Running assertions on the L2PA");
+
+        address l2paOwner = l2pa.owner();
+        require(l2paOwner == aliasedL1PAO, "checkL2PA-200");
+        require(unaliasedL1PAO == undoL1ToL2Alias(l2paOwner), "checkL2PA-201");
+    }
+
+    function applyL1ToL2Alias(address l1Address) public pure returns (address l2Address) {
+        unchecked {
+            l2Address = address(uint160(l1Address) + ALIASING_OFFSET);
+        }
+    }
+
+    function undoL1ToL2Alias(address l2Address) public pure returns (address l1Address) {
+        unchecked {
+            l1Address = address(uint160(l2Address) - ALIASING_OFFSET);
+        }
+    }
+
+    /// @notice Checks the correctness of the deployment
+    function _postCheck(Vm.AccountAccess[] memory accesses, SimulationPayload memory /* simPayload */ )
+        internal
+        view
+        override
+    {
+        console.log("Running post-deploy assertions");
+
+        checkStateDiff(accesses);
+        checkL2PAOSafe();
+        checkL2PA();
+
+        console.log("All assertions passed!");
+    }
+
+    function getCodeExceptions() internal pure override returns (address[] memory) {
+        address[] memory shouldHaveCodeExceptions = new address[](7);
+
+        shouldHaveCodeExceptions[0] = l2OutputOracleProposer;
+        shouldHaveCodeExceptions[1] = l2OutputOracleChallenger;
+        shouldHaveCodeExceptions[2] = systemConfigOwner;
+        shouldHaveCodeExceptions[3] = batchSenderAddress;
+        shouldHaveCodeExceptions[4] = p2pSequencerAddress;
+        shouldHaveCodeExceptions[5] = batchInboxAddress;
+        shouldHaveCodeExceptions[6] = aliasedL1PAO; // aliased L1PAO on op-sepolia doesn't have any code.
```

Comment:
As mentioned above, I suspect this will be the only code exception. And actually, a good check to add to the `_postCheck` would be to take the `aliasedL1PAO`, undo the alias to get the original Safe address, and check that it is a Safe on the L1 chain

Reply:
done:  https://github.com/ethereum-optimism/superchain-ops/pull/204/commits/5055a9049e26cf7cd4c847be600d41042295e377

Reply:
correction, added here: https://github.com/ethereum-optimism/superchain-ops/pull/204/commits/9cd87cbc12bbf2be0eca4f2d60032df1b424ee1d